models/org.go

```diff
@@ -419,13 +415,13 @@ func getUsersWhoCanCreateOrgRepo(e db.Engine, orgID int64) ([]*User, error) {
 		And("team_user.org_id = ?", orgID).Asc("`user`.name").Find(&users)
 }
 
-func getOrgsByUserID(sess *xorm.Session, userID int64, showAll bool) ([]*User, error) {
+func getOrgsByUserID(e db.Engine, userID int64, showAll bool) ([]*User, error) {
 	orgs := make([]*User, 0, 10)
+	sess := e.Where("`org_user`.uid=?", userID)
 	if !showAll {
-		sess.And("`org_user`.is_public=?", true)
+		sess = sess.Where("`org_user`.is_public=?", true)
```

Comment:
Although calling `s.Where().Where()` is correct, using `.And()` can be more clear in my mind.